.github/workflows/build.yml

```
@@ -142,6 +142,10 @@ jobs:
       - name: Check for unsupported C global variables
         if: github.event_name == 'pull_request'  # $GITHUB_EVENT_NAME
         run: make check-c-globals
+      - name: Check for undocumented C APIs
+        if: github.event_name == 'pull_request'
```

Why only run on PRs? Does it not work for pushes?



```suggestion
```

---

It works for pushes, but is there any point? We only run the C analyzer for PRs.

---

I think so -- if someone has gone through the bother of enabling the CI on their fork to run the tests, then they want the tests to run and find failures before opening a PR ("fail fast"). And this only takes 3s to run.